go-controller/pkg/ovn/master.go

```diff
@@ -299,6 +310,16 @@ func (oc *Controller) StartClusterMaster(masterNodeName string) error {
 		}
 	}
 
+	if oc.aclLoggingEnabled {
+		if stdout, _, err := util.RunOVNNbctl("meter-list"); err == nil && !strings.Contains(stdout, "acl-logging:") {
+			dropRate := strconv.Itoa(config.Logging.ACLLoggingRateLimit)
+			if _, _, err := util.RunOVNNbctl("meter-add", "acl-logging", "drop", dropRate, "pktps"); err != nil {
```

Comment:
@dyusupov spoke with @dceara and he does not think there is any perf impact of just always adding the meter. I think we can just remove the configuration option and always have it enabled. He did mention this though:

```
OVN meters are weird.
10:08
They're actually shared between ACLs so if one reject ACL is hit a lot you might only see logs from that one.
10:08
You need to use fair meters.. it's a new thingy. Let me check.
10:10
This: https://github.com/ovn-org/ovn/commit/880dca99eaf73db7e783999c29386d03c82093bf
```

Can you please look into if we can switch to using fair meters?

Reply:
Just quick update on this. Updated branch with removed configuration option. The --fair flag requires ovn >= 20.12. Also, after it is added we will see errors like this:
```
2021-01-14T22:39:06.243Z|00029|ofctrl|ERR|could not find meter named "acl-logging__8aa70f5b-8678-47b9-a54e-2f24317cbcc4"
2021-01-14T22:40:02.405Z|00030|ofctrl|ERR|could not find meter named "acl-logging__27b328ee-b548-4b53-af55-d344338af3b3"
```
@dceara appreciate your input on this - is this a harmless error or will it disable meter functionality?

Reply:
@dyusupov Oh, this is a bug in OVN, thanks for spotting it!  I sent a [patch to fix it](http://patchwork.ozlabs.org/project/ovn/patch/1610718079-15077-1-git-send-email-dceara@redhat.com/) upstream.

Indeed, without the OVN fix it will disable the metering functionality.